study-builder/studies/rgp/study.conf

```diff
@@ -209,26 +209,6 @@
       "delaySeconds": null,
       "dispatchToHousekeeping": false,
       "order": 1
-    },
-
-    # DSM writes "workflows" into ES and generates things like family_id, but this doesn't happen
-    # until the participant list page is visited. So we push an "update" message to DSM to force it
-    # to generate/write out these values. A good extension point we can hook into will be when they
-    # register. We also need to account for migrated users (migrated ones won't have "registered"
-    # event triggered). For the RGP study, this "registered" event happens only after user has
-    # verified their email and logs in for the first time, which is an ideal time to initiate this.
-    {
```

Comment:
This event is no longer need now that we have this new mechanism to force DSM to write/generate these values.